### ISSUE TITLE
Omniglot outputs for VHE training

### ISSUE THREAD

**Issue by rkjones4** [insperatum/vhe#1]
Hi! 

I realize this repository is likely not under active management, but I was hoping to train some VHE models on omniglot. I experimented around with the released example in the repository, and while the code was able to run without error, the sampled outputs (from even the best runs) were of very poor quality. The loss is decreasing, but at a slower rate than what I would expect, and converges to significantly higher numbers than what are reported in the paper.

One thought I have is that there might have been a breaking change in the PyTorch API that has introduced a 'silent bug', since the release of the code (e.g. flipping channel vs batch index of one of the base layers). Would it possible to provide the package versioning that was used to successfully run the scripts? Any and all points would be greatly appreciated! My fall-back plan is to replace the pixelCNN version with just a simple conv/deconv encoder-decoder set-up, but it seems from the paper that this will very likely produce poor quality outputs.

Thank you so much!
Kenny

![samples_epoch_395](https://github.com/insperatum/vhe/assets/10440535/871b8918-c286-4736-8bdd-83b44b262e82)
